package-lock.json

```diff
@@ -65931,6 +65931,7 @@
 			"version": "0.6.0",
 			"resolved": "https://github.com/woocommerce/woocommerce-blocks/files/12309320/wordpress-e2e-test-utils-playwright-0.6.0.tgz",
 			"integrity": "sha512-5lo8iRuRfUOAPMRsQteSkKyMoP1IHHu6GKye8Df+YEPz1LWXcS6YRFGI3HrONBpcaHLrJC7LVav4UdB/sBPqUw==",
+			"dev": true,
```

Comment:
Was this change intended or did it accidentally slip into this PR? 🤔

Reply:
Something's wrong in trunk, npm install will update the lock file. I reverted one, but failed to notice there was another one. Reverted both now 👍🏼